### PR TITLE
Update default JupyterHub configuration

### PR DESCRIPTION
**Pull request by jgwerner**
- Updates config to redirect users to their servers (JupyterHub.redirect_to_server = True)
- Updates default memory to 2 gigs (Spawner.mem_limit = '2G')
- Adds environment variable to set spawner memory options

### CODE REVIEW

File: ansible/group_vars/all.yml [hunk 1]
```diff
@@ -64,3 +64,6 @@ lti13_authorize_url: "{{ lti13_authorize_url_param | default('https://illumidesk
 
 # shared drive
 shared_folder_enabled: "{{ shared_folder_enabled_param | default('true') }}"
+
+# spawner
+spawner_mem_limit: "{{ spawner_mem_limit_param | default('2G') }}"
```
Comment (netoisc) [inline ansible/group_vars/all.yml:69]: A newline is missing here?

File: ansible/hosts.example [hunk 2]
```diff
@@ -97,3 +97,7 @@ all:
       # lti13_endpoint: https://illumidesk.instructure.com/api/lti/security/jwks
       # lti13_token_url: https://illumidesk.instructure.com/login/oauth2/token
       # lti13_authorize_url: https://illumidesk.instructure.com/api/lti/authorize_redirect
+
+      ## Spawner
+      # uncomment and add your preferred memory limit settings for user workspaces
+      # spawner_mem_limit: 2G
```
Comment (netoisc) [inline ansible/hosts.example:103]: same as the above comment, the newline is missing, right?

File: ansible/roles/jupyterhub/templates/env.jhub.j2 [hunk 3]
```diff
@@ -66,3 +66,5 @@ PROXY_API_URL=http://reverse-proxy:8099
 ANNOUNCEMENT_SERVICE_PORT=8889
 
 SHARED_FOLDER_ENABLED={{shared_folder_enabled}}
+
+SPAWNER_MEM_LIMIT={{spawner_mem_limit}}
```
Comment (netoisc) [inline ansible/roles/jupyterhub/templates/env.jhub.j2:70]: a newline is missing too...

File: ansible/group_vars/all.yml (hunk 1 above)
Comment (jgwerner) [reply ansible/group_vars/all.yml:69]: Perhaps? I'm not sure YAML's should have new lines or not.

**Comment by jgwerner** [comment]
@netoisc since the new line issues deal with YAML files then I'll go ahead and merge and look at the possibility of adding another linter for all YAML files in the repo, including Kubernetes manifests.

**Pull request merged**